packages/angular/package.json

```diff
@@ -7,50 +7,47 @@
     "dev": "ng build --watch",
     "start": "ng serve",
     "build": "ng build --prod",
-    "test": "echo \"Skipped: ng test\"",
+    "test": "jest",
```

Comment:
✨ Definitely can be done in a separate PR but we should add `yarn angular test` in the top level [`yarn test`](https://github.com/aws-amplify/amplify-ui/blob/main/package.json#L20) script